### PR TITLE
Lesson3

### PR DESCRIPTION
**Pull request by Alex-javac**
На проверку несколько задач)))

### CODE REVIEW

File: src/lesson3/Task20.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package lesson3;
+
+import java.util.StringTokenizer;
+
+//Имеется строка с текстом. Вывести текст, составленный из последних букв
+//всех слов.
+public class Task20 {
```
Comment (atlamax) [inline src/lesson3/Task20.java:7]: In general OK< but I would suggest better variable naming:
st -> tokenizer
str -> text
n -> character (or ch, at least)

File: src/lesson3/Task21.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package lesson3;
```
Comment (atlamax) [inline src/lesson3/Task21.java:1]: I think that, to make this comparision more real, better get Strings from StringBuffer and StringBuilder.

File: src/lesson3/Task23.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package lesson3;
```
Comment (atlamax) [inline src/lesson3/Task23.java:1]: Mistakes:
`1) </div> should be </p>, not <p>`
`2) <div class='green-text'> not processing`

File: src/lesson3/Task20.java (hunk 1 above)
Comment (Alex-javac) [reply src/lesson3/Task20.java:7]: fixed it

File: src/lesson3/Task23.java (hunk 3 above)
Comment (Alex-javac) [reply src/lesson3/Task23.java:1]: corrected

File: src/lesson3/Task21.java (hunk 2 above)
Comment (Alex-javac) [reply src/lesson3/Task21.java:1]: I dont understand

Comment (Alex-javac) [reply src/lesson3/Task21.java:1]: everything is correct by condition

**Review by atlamax — APPROVED** [review]
Well done!

**Pull request merged**